spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -631,6 +651,137 @@ public void setFailFast(boolean failFast) {
 
 	}
 
+	/**
+	 * High (and some medium) priority Streams properties and a general properties bucket.
+	 */
+	public static class Streams {
+
+		private final Ssl ssl = new Ssl();
+
+		/**
+		 * Kafka streams application.id property; default spring.application.name.
+		 */
+		private String applicationId = "${spring.application.name}";
+
+		/**
+		 * Whether or not to auto-start the streams factory bean; default false.
+		 */
+		private boolean autoStartup;
+
+		/**
+		 * Comma-delimited list of host:port pairs to use for establishing the initial
+		 * connection to the Kafka cluster.
+		 */
+		private List<String> bootstrapServers;
```

Comment:
I am confused. Is that streams specific? Yet this property doesn't indicate that's the case.

Reply:
This is similar to producer/consumer - we can add a global property and/or override at the component level - I will enhance these docs in all 3 places.